mongorestore/options.go

```diff
@@ -65,21 +65,22 @@ func (*InputOptions) Name() string {
 
 // OutputOptions command line argument long names
 const (
-	DropOption                     = "--drop"
-	DryRunOption                   = "--dryRun"
-	WriteConcernOption             = "--writeConcern"
-	NoIndexRestoreOption           = "--noIndexRestore"
-	NoOptionsRestoreOption         = "--noOptionsRestore"
-	KeepIndexVersionOption         = "--keepIndexVersion"
-	MaintainInsertionOrderOption   = "--maintainInsertionOrder"
-	NumParallelCollectionsOption   = "--numParallelCollections"
-	NumInsertionWorkersOption      = "--numInsertionWorkersPerCollection"
-	StopOnErrorOption              = "--stopOnError"
-	BypassDocumentValidationOption = "--bypassDocumentValidation"
-	PreserveUUIDOption             = "--preserveUUID"
-	TempUsersCollOption            = "--tempUsersColl"
-	TempRolesCollOption            = "--tempRolesColl"
-	BulkBufferSizeOption           = "--batchSize"
+	DropOption                      = "--drop"
+	DryRunOption                    = "--dryRun"
+	WriteConcernOption              = "--writeConcern"
+	NoIndexRestoreOption            = "--noIndexRestore"
+	IgnoreInvalidIndexOptionsOption = "--ignoreInvalidIndexOptions"
```

Comment:
I think if this is stripping invalid options, we should rename the option to "--removeInvalidIndexOptions".

Reply:
Though if we're also going to fix up the index key document, maybe we need a more inclusive name.  "convertLegacyIndexes"?